tests/one_to_one/tests.py

```diff
@@ -226,7 +226,8 @@ def test_related_object_cache(self):
         setattr(p, 'restaurant', None)
 
         # You also can't assign an object of the wrong type here
-        with self.assertRaises(ValueError):
+        msg = 'Cannot assign "%s": "Place.restaurant" must be a "Restaurant" instance.'
```

Comment:
You could use `%r` or just hard-code the repr of the class in the expected message.